ci/docker/nuxeo/templates/nuxeo-js-client-test/nxserver/config/nuxeo-js-other-repository-config.xml.nxftl

```diff
@@ -0,0 +1,34 @@
+<#escape x as x?xml>
+<?xml version="1.0"?>
+<component name="org.nuxeo.multi.repository">
+
+<#if (nuxeo.search.client.default.name)?has_content>
+  <extension target="org.nuxeo.ecm.core.search.client.repository" point="searchClient">
+    <searchClient name="repository-other" searchIndex="repository-other" repository="other" />
+  </extension>
+
+  <extension target="org.nuxeo.ecm.core.search" point="searchClient">
+    <searchClient name="repository-other" factory="org.nuxeo.ecm.core.search.client.repository.RepositorySearchClientFactory" />
+  </extension>
+
+  <extension target="org.nuxeo.ecm.core.search" point="searchIndex">
+    <searchIndex name="repository-other" searchClient="repository-other" repository="other" />
+  </extension>
+
+</#if>
```

Comment:
This part is optional as tests don't search on this SearchIndex.

Reply:
We can maybe let this configuration to have a "clean" config, and if at some point we add tests on it, it will be there.